sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/NestedSchemaPruningBenchmark.scala

```diff
@@ -151,7 +144,7 @@ object NestedSchemaPruningBenchmark extends SqlBasedBenchmark {
 
   override def runBenchmarkSuite(mainArgs: Array[String]): Unit = {
     runBenchmark(s"Nested Schema Pruning Benchmark") {
```

Comment:
Shall we parameterize this `s"Nested Schema Pruning Benchmark"` like `dataSourceName`? Currently, the content of all result files look identical (except the file name).